.dockerignore

```diff
@@ -0,0 +1,7 @@
+.git*
+.vscode
+ci-helper-app
```

Comment:
Why are we ignoring this?

Reply:
This is the default name of the binary created by user (when user runs `make build` locally). 

When user tries to build container image locally (`docker build -t containerized-ci-helper-app .`), there's no need to copy this binary to the container image, because the new binary will be built during the container image build phase.

Reply:
Very nice, thanks!